test/harmony-regression/notebook_helpers/__init__.py

```diff
@@ -0,0 +1,377 @@
+from contextlib import contextmanager
```

Comment:
There are possibly parts of this we could trim down or exclude, based on not caring much about the visual output in the current notebooks.  I'd vote for leaving it as is for now but revisiting once we revamp the existing notebooks to use the newly-developed Python lib.

Reply:
Doing the trimming down could remove a number of hefty dependencies in the current code

Reply:
HARMONY-762